manual/manual.adoc

```diff
@@ -665,3 +660,33 @@ number of cheats that you can turn to:
 | **IDBEHOLDM** | Gives the tactical survey map.
 | **IDBEHOLDL** | Gives the night vision goggles.
 |==========================
+
+<<<
+
+== Contributing to Freedoom ==
+
+Freedoom is a 
+https://en.wikipedia.org/wiki/Free_and_open-source_software[free and open source software (FOSS)]
```

Comment:
Use "free content" and the gnu.org link as the website does: https://freedoom.github.io/about.html

Reply:
Changed link to GNU's free software philosophy page

https://www.gnu.org/philosophy/free-sw.html